3_javascript/javascript1/week2/script.js

```diff
@@ -0,0 +1,87 @@
+
+// exercice full_name
+function getFullname(firstname, surname, useFormalName, isWomen ){
+
+    if( firstname === "" & surname === ""){
```

Comment:
use `&&` You should use the logical AND operator (&&) for conditional checks instead of the bitwise AND operator.